inc/class-core-sitemaps-stylesheet.php

```diff
@@ -15,25 +15,23 @@
 class Core_Sitemaps_Stylesheet {
 	/**
 	 * Renders the xsl stylesheet depending on whether its the sitemap index or not.
+	 *
+	 * @param string $type Stylesheet type. Either 'xsl' or 'index'.
 	 */
-	public function render_stylesheet() {
-		$stylesheet_query = get_query_var( 'sitemap-stylesheet' );
-
-		if ( ! empty( $stylesheet_query ) ) {
-			header( 'Content-type: application/xml; charset=UTF-8' );
-
-			if ( 'xsl' === $stylesheet_query ) {
-				// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- All content escaped below.
-				echo $this->get_sitemap_stylesheet();
-			}
+	public function render_stylesheet( $type ) {
+		header( 'Content-type: application/xml; charset=UTF-8' );
 
-			if ( 'index' === $stylesheet_query ) {
-				// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- All content escaped below.
-				echo $this->get_sitemap_index_stylesheet();
-			}
+		if ( 'xsl' === $type ) {
```

Comment:
It's odd that the valid stylesheet types are `xsl` and `index`. We should switch to `index` and `sitemap` or something

Reply:
Done.